docker-compose.yaml

```diff
@@ -128,9 +139,9 @@ services:
             - TFY_API_KEY=${TFY_API_KEY}
             - TFY_HOST=${TFY_HOST}
             - TFY_LLM_GATEWAY_URL=${TFY_LLM_GATEWAY_URL}
-            - DATABASE_URL=postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:${POSTGRES_PORT}/cognita-config
+            - DATABASE_URL=postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@cognita-db:5432/cognita-config
```

Comment:
linux systems are not able to access db via service name. #244

Reply:
Why? docker dns is supposed to work for bridge network on any OS
We should reproduce and confirm